packages/moonstone/MoonstoneDecorator/fontGenerator.js

```diff
@@ -32,11 +32,11 @@ const fonts = {
 	'ja': {
 		regular: 'LG Smart UI JP'
 	},
+	'ur': {
+		regular: ['LG Smart UI_Urdu', 'LGSmartUIUrdu'] // This needs 2 references because the "full name" differs from the "family name". To target this font file directly in all OSs we must also include the "postscript name" in addition to the "full name".
```

Comment:
I doesn't work. We need to remove `_` (`LG Smart UI Urdu`)

Reply:
Removed `_` to match the family name instead of the full name.